src/komponenter/modal/InfoModal.tsx

```diff
@@ -0,0 +1,58 @@
+import VarselTegnForModal from '@/komponenter/modal/VarselTegnForModal';
+import BEMHelper from '@/utils/bem';
+import { Modal } from '@navikt/ds-react';
+import { FunctionComponent, PropsWithChildren, useRef } from 'react';
+import './InfoModal.less';
+import LagreOgAvbrytKnapp from '../lagreOgAvbrytKnapp/LagreOgAvbrytKnapp';
+
+interface InfoModalProps {
+    open: boolean;
+    width: 'small' | 'medium';
+    confirmText?: string;
+    onConfirm?: () => any;
+    onClose: () => any;
+}
+
+const InfoModal: FunctionComponent<PropsWithChildren<InfoModalProps>> = ({
+    children,
+    width,
+    open,
+    confirmText,
+    onConfirm,
+    onClose,
+}) => {
+    const cls = BEMHelper('infomodal');
+    const ref = useRef<HTMLDialogElement>(null);
+
+    return (
+        <div className={cls.className}>
+            <Modal
+                ref={ref}
+                open={open}
+                onClose={onClose}
+                width={width || 'medium'}
+                aria-label="Informasjonsdialog"
+                className={cls.element('modal-container')}
+            >
+                <Modal.Header>
+                    <VarselTegnForModal className={cls.element('varsel')} width={'80px'} height={'80px'} />
+                </Modal.Header>
+                <Modal.Body>
+                    {children}
+                    {onConfirm ? (
+                        <LagreOgAvbrytKnapp
+                            className={cls.element('knapper-container')}
+                            lagreFunksjon={onConfirm}
+                            lagretekst={confirmText ? confirmText : 'Bekreft'}
+                            avbryt={() => {
+                                ref.current?.close();
+                                onClose();
```

Comment:
Hvordan funker dette? Er det ikke `open` propen som styrer om modalen er åpen eller ikke? 🤔

Reply:
Det er vel kanskje et [dialog](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) element som Aksel bruker? Er litt rart å ha to måter å åpne/lukke på læll...

Reply:
Riktig at aksel bruker dialog-element ja! Men jeg gjorde uansett feil her slik at onClose kalles dobbelt: 9949e9ca113b442c813a8907841d8716c2906be3